frontend/src/toolbar/button/ToolbarButton.js

```diff
@@ -0,0 +1,268 @@
+import './ToolbarButton.scss'
+
+import React, { useRef, useEffect } from 'react'
+import { useActions, useValues } from 'kea'
+import { useLongPress } from 'lib/hooks/useLongPress'
+import { CloseOutlined, ProfileOutlined, SearchOutlined, FireFilled } from '@ant-design/icons'
+import { Logo } from '~/toolbar/assets/Logo'
+import { Circle } from '~/toolbar/button/Circle'
+import { toolbarButtonLogic } from '~/toolbar/button/toolbarButtonLogic'
+import { heatmapLogic } from '~/toolbar/elements/heatmapLogic'
+import { dockLogic } from '~/toolbar/dockLogic'
+import { toolbarLogic } from '~/toolbar/toolbarLogic'
+import { getShadowRoot } from '~/toolbar/elements/utils'
+import { ChangingText } from '~/toolbar/button/ChangingText'
+import { elementsLogic } from '~/toolbar/elements/elementsLogic'
+
+const quarters = { ne: 0, nw: 1, sw: 2, se: 3 }
+
+function getQuarterRotation({ itemCount, index, quarter, padding: inputPadding }) {
+    const padding = typeof inputPadding !== 'undefined' ? inputPadding : 90 / itemCount
+    const angle = quarter * 90 + (itemCount === 1 ? 45 : padding / 2 + ((90 - padding) / (itemCount - 1)) * index)
+    return -angle
+}
+
+function reverseQuarter(quarter) {
+    return (quarter[0] === 'n' ? 's' : 'n') + (quarter[1] === 'e' ? 'w' : 'e')
+}
+
+const walkLength = 20
+const hedgehogWalk = () =>
+    Array.from(Array(walkLength)).map((_, i) => [`${'.'.repeat(walkLength - i)}🦔${'.'.repeat(i)}`, 60])
+
+const loggedOutLines = hedgehogWalk().concat([
```

Comment:
I think this could just be replaced with a static "Click to authenticate"

Reply:
Just like this? 
![image](https://user-images.githubusercontent.com/53387/84771284-6d9d2280-afd9-11ea-8400-64c7ce6cf05b.png)

This is now pushed.